package.json

```diff
@@ -52,6 +51,7 @@
     "dependencies": {
         "amqplib": "^0.5.5",
         "axios": "^0.19.2",
+        "chai": "^4.2.0",
```

Comment:
I don't think it's a good idea to use it in prod.
Just use ``assert`` module. It's a best practice.

Reply:
Why? Shouldn't we be consistent with how we test to keep things simple? This is a module to aid testing.

Reply:
True to that.